HISTORY.md

```diff
@@ -6,6 +6,9 @@
 * Fixed an optimistic transaction validation bug caused by DBImpl::GetLatestSequenceForKey() returning non-latest seq for merge (#10724).
 * Fixed a bug in iterator refresh which could segfault for DeleteRange users (#10739).
 
+### New Features
+* Add option `preserve_internal_time_seconds` for tiered storage migration, which can start collecting the preserving the time information for the new data, but won't enable the tiering feature (`preclude_last_level_data_seconds`) yet. The time information can be used to decide the temperature of the existing data when the tiering is enabled later.
```

Comment:
preserve_internal_time_seconds can potentially be used for more accurate oldest ancestor time estimation, right? So maybe explain it more in a boarder way.

Reply:
The feature hasn't been implemented yet (we abandoned #8185 because based on test, there's limited improvement with random dataset).

Reply:
I know. I mean just be vague about what the feature does, and not limit to tiered storage. Even now, the feature can be used for manual checking the age of the keys.